internal/release/promote/promotion.go

```diff
@@ -75,10 +76,12 @@ func NewDefaultPromotion(catalogDir, gitHubOrganization, commitTemplate, pullReq
 		commitTemplate,
 		pullRequestTemplate,
 		func(proj *v1alpha1.Project) string {
-			if proj.Spec.Repository != "" {
-				return proj.Spec.Repository
-			}
-			return fmt.Sprintf("%s/%s", gitHubOrganization, proj.Name)
+			return func(proj *v1alpha1.Project) string {
```

Comment:
Is this an editing mistake? Why does the function return the results of invoking an IIFE here?

Reply:
Isn't you who told me to use more IIFEs? I thought that one was particularly nice.

(Kidding, of course, probably an oversight after some refactoring I made, as previously I had some function currying going on, but no more)